app/Config/Schema/schema.xml

```diff
@@ -1398,6 +1398,7 @@
     </field>
     <field name="revision" type="I" />
     <field name="deleted" type="L" />
+    <field name="form_hide" type="L" />
```

Comment:
Change to `form_hidden` to be consistent with the existing `hidden` column

Reply:
Done